infra/cli/cli.go

```diff
@@ -14,25 +14,38 @@ func NewCLI() CLI {
 	return CLI{}
 }
 
-func (cli *CLI) Execute(args []string) (usecases.ScrapingCmd, error) {
+func (cli *CLI) Execute(args []string) ([]usecases.ScrapingCmd, error) {
```

Comment:
I would suggest that the cmd returned from cli.Execute() should be left as a single cmd but includes a list of urls. Rather than reusing the command to trigger the usecase, construct multiple usecase-specific commands from the initial cli-command. 

basically introducing a CliCmd and not reusing the ScrapingCmd (which is already a different package) and the cli should not know anything about the user anyhow. Sorry, I was lazy and did not keep the boundaries right.